src/app/player/components/player.component.scss

```diff
@@ -25,6 +25,10 @@
   .loser svg {
     outline-color: red;
   }
+
+  .winner svg {
+    outline-color: green;
+  }
```

Comment:
Why is this (and the class application in the template) necessary? The default outline color is green, defined in line 20.
If you knew it and wanted to make it more explicit I am ok with it, but remove the double color declaration from the mentioned line please. 
I for one prefer the old way :)

Reply:
<img src="https://media1.giphy.com/media/K0Hy2NwI8IXZK/giphy.gif"/>